packages/combobox/src/Combobox.story.tsx

```diff
@@ -113,15 +113,35 @@ export default StoryMeta({
   },
 });
 
-const ComboboxOptions = [
+const getComboboxOptions = (withGlyphs = true) => [
   <ComboboxOption
     key="apple"
     value="apple"
     displayName="Apple"
-    data-testid="hii"
+    data-testid="test-id"
+    description="Do I keep the doctor away?"
+    // eslint-disable-next-line no-console
+    onClick={(event, value) => console.log(event.currentTarget, value)}
```

Comment:
nit: let's log the event itself, because it'll be easier for developers to understand the function's signature